src/main/groovy/wooga/gradle/github/base/GithubBasePluginConvention.groovy

```diff
@@ -17,49 +17,53 @@
 
 package wooga.gradle.github.base
 
+import com.wooga.gradle.PropertyLookup
+
 /**
  * Constant values for github plugin.
  */
 class GithubBasePluginConvention {
+
     /**
      * Gradle property name to set the default value for {@code username}.
      * @value "github.username"
      * @see GithubSpec#getUsername()
      */
-    static final String GITHUB_USER_NAME_OPTION = "github.username"
+    static final PropertyLookup userName = new PropertyLookup(null, "github.username", null)
```

Comment:
You need to keep the old fields since removing them is an API break.

Reply:
Ah, my apologies.

Reply:
You could use the static Fields so you don't repeat the values. But that doesn't really matter

Reply:
I can do that much.